app/controllers/bins_controller.rb

```diff
@@ -4,7 +4,7 @@ class BinsController < ApplicationController
   active_tab "inventory"
 
   def index
-    @bins = Bin.includes(:bin_location, :items).all.to_a
+    @bins = Bin.not_deleted.includes(:bin_location, :items).all.to_a
```

Comment:
I see where you are coming from, and yeah, it probably could have been done in it's own ticket/PR, though I came to it because it was required as part of this ticket (and I would not have done it if it hadn't been needed for what I was doing), so it didn't occur to me to break it out at the time.